tests/utils/helper/helper.go

```diff
@@ -25,6 +26,13 @@ func Parse(data []byte) map[string]interface{} {
 	return object
 }
 
+func GetJsonFromPath(path string, filename string) map[string]interface{} {
+	combinedFilePath := path + "/" + filename
```

Comment:
how about use `path.Join` in case somebody running code on Windows

Reply:
path.join giving an error on the `helper.go` file, it does not want to import the path package

Reply:
It's not related to the import, you named the parameter as `path`, thus once you imported another path, you will get an error

Reply:
ahh I see
ok, will change that